using-funds/cash-advances.md

```diff
@@ -0,0 +1,9 @@
+# Cash Advances
+
+After being approved for a [grant](https://github.com/bridgefoundry/operations/tree/master/grants), Independent Communities that can't afford to get reimbursed after the fact can receive funds via a cash advance.
+
+The big downside of a cash advance is that you will need to send back any extra money that you did not spend. If you can afford to do so, it is best to low ball your estimate and then get reimbursed for a the small amount that you go over.
```

Comment:
Best for who?

Reply:
Best for the volunteer, I was figuring. It's more work for them to have to send back the extra money. What if we change the phrasing to something like "To avoid the extra work of sending back the money, if you can afford to do so we recommend low balling your estimate and then getting reimbursement for the small amount you go over."  We could also leave out the comment about it being best and just clarify that if they go slightly over the amount advanced we'll reimburse the remaining amount when they submit their report. What do you think?

Reply:
I think sending money back to bridge foundry is probably going to be easier than getting a reimbursement.  Paypal vs. expensify?

But maybe just say if they're under they have to send the extra back and if over they can submit a reimbursement.